Watchman.Discord/Areas/Reactions/Services/ReactionsService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+using Watchman.Discord.Areas.Reactions.Contexts;
+
+namespace Watchman.Discord.Areas.Reactions.Services
```

Comment:
czemu twierdzisz że powinniśmy to tworzyć osobno za każdym razem, kiedy tworzymy bota za pomocą tego frameworka?

Reply:
`Areas.Reactions.Services`
A co gdybyśmy chcieli użyć tego w Areas.Roles?
Bo chcielibyśmy dodać opcje przydzielania roli za pomocą reakcji

Reply:
Nie rozumiem co masz na myśli mówiąc, że tworzyć osobno. A gdybyśmy chcieli tego użyć gdzie to prostu się użyj i tyle.

Reply:
no tylko że wtedy tworzy sie tak zwany spaghetti code